python/core/qgsvectorlayer.sip

```diff
@@ -685,6 +685,11 @@ class QgsVectorLayer : QgsMapLayer, QgsExpressionContextGenerator
     virtual QString getStyleFromDatabase( const QString& styleId, QString &msgError /Out/ );
 
     /**
+     * Will delete the named style corresponding to style id provided from the database
+     */
+    virtual void deleteStyleFromDatabase( const QString& styleId, QString &msgError /Out/ );
```

Comment:
+1 @m-kuhn I generally prefer an explicit bool as return success/fail value instead of hiding error in the msgError value Null or !Null, btw I don't have a stronger opinion on that.
I agree that if msgError is usdes, it can be the return value of the function. I suppose @jgrocha followed the same style as in getStyleFromDatabase where the return is the style string.